.gitignore

```diff
@@ -58,3 +58,5 @@ target/
 
 # pyLCI
 /config.json 
+/pylci.log*
+do_not_load
```

Comment:
do_not_load, as of now, can't be added to gitignore - those files are used to exclude loading of example apps, as well as hardware-specific apps, in stock pyLCI install